app/views/wpcc/shared/_your_details.html.erb

```diff
@@ -1,14 +1,12 @@
 <section class="section section--details details">
   <h2 class="section__heading details__heading">1. <%= t('wpcc.details.title') %>
-    <span class="section__heading-summary"><%= t('wpcc.details.section__heading.age', age: session[:age]) %>,
-      <%= t('wpcc.details.section__heading.gender_' + session[:gender]) %>,
-      <%= t('wpcc.details.section__heading.salary_' + session[:salary_frequency], salary: session[:salary]) %>,
-      <%= t('wpcc.details.section__heading.contribution_' + session[:contribution_preference]) %></span>
+    <span class="section__heading-summary">
+      <%= message_presenter.your_details_summary(session) %>
+    </span>
```

Comment:
For reasons connected with the requirement to print the user's results we are using CSS generated content to render the parentheses here. Consequently can we remove any whitespace between the <span></span> tags and the content here?